### ISSUE TITLE
rpminspect incorrectly complains about added files

### ISSUE THREAD

**Issue by hanzz** [rpminspect/rpminspect#708]
Originally reported downstream (OSCI-3116):

After rebasing libvirt, rpminspect reports the following failures for all architectures:

```
New security-related file `/usr/share/polkit-1/actions/org.libvirt.api.policy` added on ppc64le requires inspection by the Security Team
New security-related file `/usr/share/polkit-1/actions/org.libvirt.unix.policy` added on ppc64le requires inspection by the Security Team
```

But this is nonsense as the files are present in both builds. In
other words they were not newly added. Actually they have been in the same
location for ages.
